git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -104,8 +110,10 @@ $ cd ~/Desktop
 $ mkdir planets
 $ cd planets
 ```
+The `cd` command means **change directory**, and `mkdir` tells the computer to **make** a **directory** with the name "planets." Once we have a `planets` directory, we use `cd` open that directory.
```

Comment:
even though we mentioned it, maybe have a warning box here with "seeing this error?  If you're on Windows you probably need to switch over to using Git Bash!"

Reply:
Added as a "help" box